main.c

```diff
@@ -1 +1,5 @@
 #import main.h;
+
+void main() {
+    printf("Hello World!");
```

Comment:
Hey Lorenzo, I created a PR few days ago but you probably haven't seen it so I self approved.  I added a define in main.h called WORLD.  please use this define here in the printf.

Reply:
I'm on ir

Reply:
Done! @JacopoMangiavacchi